otel/otel.go

```diff
@@ -0,0 +1,171 @@
+package otel
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	otelsdk "go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp"
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutmetric"
+	"go.opentelemetry.io/otel/metric"
+	sdkmetric "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.26.0"
+)
+
+// Config defines the Otel Wrapper configuration for metrics
+type Config struct {
+	ServiceName               string
+	Debug                     bool
+	ExporterType              string
+	ExporterEndpoint          string
+	ExporterTimeout           string
+	ExporterCertificate       string
+	ExporterClientCertificate string
+	ExporterClientKey         string
```

Comment:
I think that there's so much thing in the same Config struct and that we need to make something like:
```go
type ExporterConfig {
  	Type              string
	Endpoint          string
	Timeout           string
	Certificate       string
	ClientCertificate string
	ClientKey         string
}

type ReaderConfig [
  Interval int
}

type Config struct {
  ServiceName string
  Debug bool
  Exporter ExporterConfig
  Reader ReaderConfig
}
```

This should also use environment tag to be correctly initialized from environment variables, but let's make that in another PR